crowbar_framework/Gemfile

```diff
@@ -58,7 +58,7 @@ gem "rack-mini-profiler", "~> 0.9",
 
 group :development, :test do
   gem "brakeman", "~> 2.6"
-  gem "rspec-rails", "~> 2.14"
+  gem "rspec-rails", "~> 3.1"
```

Comment:
I'm not against reusing the gems through rpm at all (the rpm packaged version is not shipping Gemfile anyway so your point ins mood) but I would like to ensure that the dev environment is reasonably similar to the production environment to avoid accidental breakages from slipping in.